### ISSUE TITLE
Typo in macros.cfg?? Should it be M400e or M400??

### ISSUE THREAD

**Issue by smackoz** [Rat-OS/RatOS-configuration#67]
Is there a typo in the GCODE macro for LOAD_FILAMENT??
M400e or M400??
```
[gcode_macro LOAD_FILAMENT]
.
.
.
\# Wait for purge to complete
M400e <<<< Is this correct or should it be M400 ??
M117 Filament loaded!
RESPOND MSG="Filament loaded!"
RESTORE_GCODE_STATE NAME=load_state
```

**Comment by miklschmidt** [comment]
That is indeed a typo. Thanks for reporting.

**Comment by miklschmidt** [comment]
Fixed in 2.0
